From f4c8e2cdacc0ca8574aa632c7436114bc416b6c4 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Sat, 13 Dec 2014 12:58:06 +0800
Subject: [PATCH] arm64: rockchip: rockchip_defconfig enable some ARM64 feature

+DISABLE_CPU_SCHED_DOMAIN_BALANCE
+SCHED_HMP
+HMP_VARIABLE_SCALE
+HMP_FREQUENCY_INVARIANT_SCALE
+SCHED_HMP_LITTLE_PACKING
+ARM_BIG_LITTLE_CPUFREQ
+ARM_DT_BL_CPUFREQ
+ARM64_CPUIDLE
+SERIAL_ROCKCHIP_CONSOLE
+RK30_CAMERA_ONEFRAME

Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index a3625009d62a..b1ea9437c3d7 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -34,6 +34,11 @@ CONFIG_ARCH_ROCKCHIP=y
 CONFIG_BLOCK_RKNAND=y
 CONFIG_SMP=y
 CONFIG_SCHED_MC=y
+CONFIG_DISABLE_CPU_SCHED_DOMAIN_BALANCE=y
+CONFIG_SCHED_HMP=y
+CONFIG_HMP_VARIABLE_SCALE=y
+CONFIG_HMP_FREQUENCY_INVARIANT_SCALE=y
+CONFIG_SCHED_HMP_LITTLE_PACKING=y
 CONFIG_PREEMPT=y
 CONFIG_ARMV7_COMPAT=y
 CONFIG_KSM=y
@@ -57,7 +62,10 @@ CONFIG_CPU_FREQ_GOV_POWERSAVE=y
 CONFIG_CPU_FREQ_GOV_USERSPACE=y
 CONFIG_CPU_FREQ_GOV_ONDEMAND=y
 CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
+CONFIG_ARM_BIG_LITTLE_CPUFREQ=y
+CONFIG_ARM_DT_BL_CPUFREQ=y
 CONFIG_CPU_IDLE=y
+CONFIG_ARM64_CPUIDLE=y
 CONFIG_NET=y
 CONFIG_PACKET=y
 CONFIG_UNIX=y
@@ -315,6 +323,7 @@ CONFIG_ROCKCHIP_REMOTECTL_PWM=y
 # CONFIG_VT is not set
 # CONFIG_LEGACY_PTYS is not set
 CONFIG_SERIAL_ROCKCHIP=y
+CONFIG_SERIAL_ROCKCHIP_CONSOLE=y
 # CONFIG_HW_RANDOM is not set
 CONFIG_I2C=y
 CONFIG_I2C_CHARDEV=y
@@ -366,6 +375,7 @@ CONFIG_MEDIA_RC_SUPPORT=y
 CONFIG_MEDIA_CONTROLLER=y
 CONFIG_VIDEO_V4L2_SUBDEV_API=y
 CONFIG_V4L_PLATFORM_DRIVERS=y
+CONFIG_RK30_CAMERA_ONEFRAME=y
 CONFIG_FB=y
 CONFIG_BACKLIGHT_LCD_SUPPORT=y
 # CONFIG_LCD_CLASS_DEVICE is not set
-- 
2.35.3

